lesson_16/task1/developer.py

```diff
@@ -0,0 +1,6 @@
+from PY_homework_hillel.lesson_16.task1.employee import Employee
+
+
+class Developer(Employee):
+
+    programming_language:str
```

Comment:
This attribute is better to be with private access.

Reply:
Thanks, Noted that it is better to use private variables in Classes